lib/ansible/modules/network/nxos/nxos_static_route.py

```diff
@@ -144,9 +148,30 @@ def remove_route_command(prefix, w):
     return 'no ip route {0} {1}'.format(prefix, w['next_hop'])
 
 
-def set_route_command(prefix, w):
+def get_configured_track(module, ctrack):
+    check_track = 'Track {0}'.format(ctrack)
+    track_exists = False
+    command = 'show track | awk "/Track/" | no-more'
```

Comment:
I tried to test your PR. `awk` is not a valid command for my device? It will fail.
I am on a 7K device. Can you run just the command `show track` and then extract track information?